src/styles/utils/_variables.scss

```
@@ -0,0 +1,10 @@
+$card-square: 98px;
+$card-gapNborder: 1px;
```

it will be better not to make this variable double-purposed 
```suggestion
$cell--gap: 1px;
$cell-border: 1px;
```